src/components/molecules/global-nav/global-nav.njk

```diff
@@ -0,0 +1,11 @@
+<nav class="global-nav" aria-labelledby="global-nav-label">
+	<span id="global-nav-label" class="screen-reader-text">global navigation</span>
+	<ul class="global-nav__list">
+		<li class="global-nav__list-item global-nav__list-item--first">
+			<a class="global-nav__link" rel="external" href="https://platform.coop">{% render '@svg', {svg: 'pcc', viewBox: '0 0 32 32'}, true %} Platform Cooperativism Consortium</a>
+		</li>
+		<li class="global-nav__list-item">
+			<a class="global-nav__link" rel="current" href="https://resources.platform.coop/">Resource Library</a>
```

Comment:
`aria-current` is probably needed here.

Reply:
I'm using `aria-current="true"` as opposed to `aria-current="page"`. For reference:

> `true` represents the current item within a set

https://tink.uk/using-the-aria-current-attribute/